template-compose/app/src/main/java/co/nimblehq/template/compose/extensions/ContextExt.kt

```diff
@@ -0,0 +1,7 @@
+package co.nimblehq.template.compose.extensions
+
+import android.content.Context
+import android.widget.Toast
+
+fun Context.showToast(message: String) =
+    Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
```

Comment:
(Optional) Ditto here 😄 
```suggestion
fun Context.showToast(message: String, length: Int = Toast.LENGTH_SHORT) =
    Toast.makeText(this, message, length).show()
```